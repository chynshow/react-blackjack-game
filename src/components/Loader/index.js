import React from 'react';

const Loader = () => <div className='c-loader' />;

export default Loader;
