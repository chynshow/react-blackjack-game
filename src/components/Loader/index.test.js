import React from 'react';
import Loader from './index';
import { shallow } from 'enzyme';

it('render Loader component', () => {
  const c = shallow(<Loader />);
  expect(c).toMatchSnapshot();
});
