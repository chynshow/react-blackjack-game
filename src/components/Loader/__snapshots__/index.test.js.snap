// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Loader component 1`] = `
<div
  className="c-loader__container"
>
  <div
    className="c-loader"
  >
    <div
      className="c-loader__cube1"
    />
    <div
      className="c-loader__cube2"
    />
  </div>
</div>
`;
